test/organization/test.sh

```diff
@@ -37,15 +38,40 @@ function sql() {
     fi
 
     if [ -n "${ROLE}" ]; then
-      log_debug "Executing query '${QUERY}' as ${ROLE}"
-      RESULT=`${CMD} -U "${ROLE}" ${DATABASE} -c "${QUERY}" -A -t`
+        log_debug "Executing query '${QUERY}' as ${ROLE}"
+        RESULT=`${CMD} -U "${ROLE}" ${DATABASE} -c "${QUERY}" -A -t 2>"${ERROR_OUTPUT_FILE}"`
```

Comment:
I wanted to check permission granting for tests like `test_group_permission_functions_cant_be_used_by_normal_members`. If you try to grant something you can't warnings are outputted to stderr. Take a look at the following lines. I think this was the easiest way to solve this (and Travis has worked), but maybe you have a better solution :)